libs/novui/src/json-schema-components/JsonSchemaForm.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { RJSFSchema } from '@rjsf/utils';
+import validator from '@rjsf/validator-ajv8';
+import Form from '@rjsf/core';
+
+import { CheckboxWidget, SelectWidget, InputWidget } from './widgets';
+
+import { AddButton, MoveDownButton, RemoveButton, MoveUpButton } from './templates/IconButton';
+import { ArrayFieldItemTemplate, ArrayFieldTemplate, ArrayFieldTitleTemplate } from './templates/ArrayFieldTemplate';
+import { ObjectFieldTemplate } from './templates/ObjectFieldTemplate';
+
+export const JsonSchemaForm = ({
+  schema,
+  formData,
+  onChange,
+}: {
+  schema: RJSFSchema;
+  formData: any;
+  onChange?: (data: any) => void;
+}) => {
+  const widgets = {
+    CheckboxWidget: CheckboxWidget,
+    SelectWidget: SelectWidget,
+    TextWidget: InputWidget,
+  };
+
+  const uiSchema = {
+    'ui:globalOptions': { addable: true, copyable: false, label: true, hideError: true, orderable: true },
+    'ui:options': {
+      hideError: true,
+      submitButtonOptions: {
+        norender: true,
+      },
+    },
+  };
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** These are static, so please define them outside of the component and use constant capitalization `SCREAMING_CAMEL_CASE`‏